src/iuse.cpp

```diff
@@ -2396,10 +2390,24 @@ int iuse::crowbar( player *p, item *it, bool, const tripoint &pos )
     }
 
     /** @EFFECT_STR speeds up crowbar prying attempts */
-
-    p->mod_moves( -( std::max( 20, ( difficulty * 20 ) - ( ( p->str_cur ) * 5 ) ) ) );
+    p->mod_moves( -std::max( 20, difficulty * 20 - p->str_cur * 5 ) );
     /** @EFFECT_STR increases chance of crowbar prying success */
 
+    // Doors need PRY 2 which is on a crowbar, crates need PRY 1 which is on a crowbar
+    // & a claw hammer. 
+    // The iexamine function for crate supplies a hammer object.
+    // So this stops the player (A)ctivating a Hammer with a Crowbar in their backpack
+    // then managing to open a door.
+    const int pry_level = it->get_quality( quality_id( "PRY" ) );
+
+    if( pry_level < pry_quality ) {
+        p->add_msg_if_player( _( "You can't get sufficient leverage to open that with your %s." ), it->tname() );
+        return 0;
+    }
+
+    // For every level of PRY over the requirement, remove n from the difficulty (so -2 with a PRY 4 tool)
+    difficulty -= ( pry_level - pry_quality );
```

Comment:
Seems like this should happen before you `p->mod_moves()`.  It should go faster with a better tool as well.

Reply:
Moved this around, there's a small penalty for trying & failing to pry because of insufficient prying quality. Reducing the difficulty has the side-effect of making it faster.